config/application.yml.example

```diff
@@ -36,6 +39,9 @@ production:
   logins_per_ip_limit: '20'
   logins_per_ip_period: '8'
   newrelic_license_key: 'xxx'
+  otp_delivery_blocklist_bantime: '10'
```

Comment:
When you add phone confirmation, you might want to change this to `sms_delivery`.